cmd/ansible.go

```diff
@@ -148,3 +173,70 @@ var playCmd = &cobra.Command{
 		return nil
 	},
 }
+
+//usage: jarvis ansible list
+//returns:
+// --group(bool) >> list groups name
+// --hosts(string) >> hosts by group
+var inventoryCmd = &cobra.Command{
+	Use:   "list",
+	Short: "Query inventory",
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		//checking exclusivity
+		if ListGroup && HostGroupName != "" {
+			return fmt.Errorf("--group and --hosts are mutually exclusive")
+		}
+
+		return nil
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		envsPath := viper.GetString("environments.path")
+
+		allInventories, err := environment.GetFullPathInventoriesFromEnvironments(envsPath, *environments)
+		if err != nil {
+			return err
+		}
+
+		var invReaders []io.Reader
+		for _, invs := range allInventories {
+			for _, inv := range invs {
+				f, err := os.Open(inv)
+				if err != nil {
+					return err
+				}
+				invReaders = append(invReaders, bufio.NewReader(f))
```

Comment:
in debug mode, I'd print the  loaded inventory files

Reply:
yes, I'll add it